recipe/meta.yaml

```diff
@@ -1,31 +1,35 @@
-{% set name = "pysal" %}
-{% set version = "2.0.0" %}
+{% set version = "2.1.0" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: pysal
+  version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e8dc75e48a6a16e89f295f7a25868671837862cb7d4862a3a171ddc80d07a23f
+  url: https://pypi.io/packages/source/p/pysal/pysal-{{ version }}.tar.gz
+  sha256: 3c21c58ebca187f2c2d6b83b624739627a1201f6bc7b80d6f970762e76149911
 
 build:
   number: 0
   noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   host:
     - python >=3.6
     - pip
   run:
     - python >=3.6
-    - descartes
-    - matplotlib
-    - palettable
+    - scipy
     - pandas
-    - scipy >=0.11
     - seaborn
+    - matplotlib
+    - descartes
+    - palettable
+    - scikit-learn
+    - geopandas
+    - deprecated
+    - rtree
+    - tqdm
 
```

Comment:
should [`zstd`](https://github.com/pysal/pysal/blob/5061e08972f1aad0bd1878f1a2f9d576fcdc78f3/ci/travis/37-latest-conda-forge.yaml#L22) be inlcuded here?

Reply:
I just added but the truth is that it "should not." Locally, when I try to re-create this build, it is pulled as expected with `fiona`. It is puzzling that this is not working as expected on the CIs.

Reply:
This problem has cropped up within the last month or so, correct?

Reply:
OK. I think I found out the problem but I'm not sure what is causing it though. The local build uses `libtiff 4.0.10-h57b8799_1003`, that has `zstd`, but somehow the CIs are pulling `libtiff 4.0.10-h648cc4a_1001`, which was not built with `zstd`.

I could not reproduce that even by downgrading my `conda` version and removing `strict`. Somehow, locally, I always get the right `libtiff`. I know `conda`'s solver sometimes prefer "less" dependencies, so the version without `zstd` may be preferred. What I cannot explain is why I cannot reproduce it locally.

I think it is safe to more the old `libtiff` without `zstd` to broken, that should "fix" things downstream and avoid more headaches. It is also probably safer to explicitly add `zstd` to `gdal` to avoid future headaches.

Pinging @msarahan here just for to share the fun of packaging :wink:.